server/seeders/20170324125925-RolesSeeder.js

```diff
@@ -0,0 +1,32 @@
+const faker = require('faker');
+
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.bulkInsert('Roles', [
```

Comment:
Can DRY this one up as well by mapping over an array of role `name`s and producing `Role`s. 🙂